hw/ip/flash_ctrl/flash_ctrl.core

```diff
@@ -10,6 +10,8 @@ filesets:
     depend:
       - lowrisc:ip:tlul
       - lowrisc:prim:all
+      - lowrisc:prim:ram_1p
```

Comment:
I cannot find where prim_ram_1p is used in code inside `hw/ip/flash_ctrl`. Is this really needed, or a dependency which is actually missing somewhere else?

Reply:
Yes it is needed, otherwise the compilation fails. It is actually used by `prim_generic_flash.sv`. I added it as a dependency of `prim_flash` instead.